manifests/opendatahub/base/buildconfigs/vizualization-server.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: visualization-server
+spec:
+  output:
+    to: 
+      kind: ImageStreamTag
+      name: 'visualization-server:1.3.1'
+  resources:
+    limits:
+      cpu: "4"
+      memory: 8Gi
+    requests:
+      cpu: "4"
+      memory: 8Gi
+  completionDeadlineSeconds: 1800
+  successfulBuildsHistoryLimit: 1
+  failedBuildsHistoryLimit: 1
+  strategy:
+    type: Docker
+    dockerStrategy:
+      from:
+        kind: ImageStreamTag
+        name: 'tensorflow:py3.8-cuda-11.4.2-2'
+      dockerfilePath: backend/Dockerfile.persistenceagent
+  postCommit: {}
+  source:
+    type: Git
+    git:
+      uri: 'https://github.com/opendatahub-io/ml-pipelines'
+      ref: master
```

Comment:
When we move this over into the release branch we will need to update the branch ref too